src/main/webapp/WEB-INF/view/conversations.jsp

```diff
@@ -85,7 +88,7 @@
       for(Conversation conversation : conversations){
     %>
       <a href="/chat/<%= conversation.getTitle() %>">
-        <%= conversation.getTitle() %></a>
+        <p>   <%= conversation.getTitle() %></a>   <p>
```

Comment:
nit: right now you have `<a><p></a></p>` where the opening and closings are mixed. You probably want `<a><p></p></a>` or better yet `<p><a></a></p>`